grizli/prep.py

```diff
@@ -5571,7 +5796,7 @@ def extract_fits_log(file='idk106ckq_flt.fits', get_dq=True):
     log['chips'] = []
 
     if get_dq:
-        idx = np.arange(1014**2, dtype=np.int32).reshape((1014, 1014))
+        idx = np.arange(2048**2, dtype=np.int32).reshape((2048, 2048))
```

Comment:
This doesn't seem to be in a JWST logic block?  Should the appropriate size be determined somewhere above?

Reply:
I added the if statement for now, but I hope to get rid of "isJWST" and the hard coded sizes eventually: e7d4f097feecc58bf9d182979f00c7a13b083fe1